components/org.wso2.carbon.identity.oidc.session/pom.xml

```diff
@@ -115,6 +115,10 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Do we need this dependency?

Reply:
For PowerMockIdentityBaseTest , we need this dependency